docs/config.json

```diff
@@ -1290,15 +1289,15 @@
       "last_report": "August 9th, 2022"
     },
     "teleport": {
-      "major_version": "11",
-      "version": "11.0.3",
+      "major_version": "12",
+      "version": "12.0.0",
```

Comment:
There was a publishing issue with 12.0.0, so these need to be updated to 12.0.1

Reply:
```suggestion
      "version": "12.0.1",
```